packages/flutter/test/widgets/system_context_menu_test.dart

```diff
@@ -728,4 +728,25 @@ void main() {
     expect(diagnosticsNodes.first.name, 'title');
     expect(diagnosticsNodes.first.value, title);
   });
+
+  test(
+    'can get the IOSSystemContextMenuItemData representation of an IOSSystemContextMenuItemLiveText',
+    () {
+      const IOSSystemContextMenuItemLiveText item = IOSSystemContextMenuItemLiveText();
+      const WidgetsLocalizations localizations = DefaultWidgetsLocalizations();
+      final IOSSystemContextMenuItemData data = item.getData(localizations);
+      expect(data, isA<IOSSystemContextMenuItemDataLiveText>());
+      expect((data as IOSSystemContextMenuItemDataLiveText).title, isNull);
+    },
+  );
+
+  test('IOSSystemContextMenuItemDataLiveText debugFillProperties', () {
+    const String? title = null;
+    const IOSSystemContextMenuItemDataLiveText item = IOSSystemContextMenuItemDataLiveText(title: title);
```

Comment:
You're getting an analyzer failure here because `null` is the default value, so you don't need to pass it.